demisto_sdk/commands/common/errors.py

```diff
@@ -464,7 +464,9 @@ def invalid_deprecated_integration_display_name():
     @staticmethod
     @error_code_decorator
     def invalid_deprecated_integration_description():
-        return 'The description of all deprecated integrations should start with "Deprecated.".'
+        return 'The description of all deprecated integrations should start with "Deprecated."' \
+               ' and contain both the phrases: "Use the", "integration instead"' \
+               ' or the phrase: "No available replacement".'
```

Comment:
I am not sure if we want to permit this level of freedom. maybe we should print the 3-4 strict examples from the issue in the message and enforce all of dose possibilities with a strict regex?

Reply:
I implemented it more loosely on purpose, I see no need for the stricter version, but your call.

Reply:
I'm with Yuval on this one. we should use a strict format